# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xdg-sound-theme/hotfix-autogen.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xdg-sound-theme-0.8/autogen.sh.vanilla	2026-09-17 19:31:31.867698130 +0300
+++ xdg-sound-theme-0.8/autogen.sh	2026-09-17 19:31:55.574524637 +0300
@@ -23,8 +23,8 @@
 
 if test x$NOCONFIGURE = x; then
   echo Running $srcdir/configure $conf_flags "$@" ...
-  $srcdir/configure $conf_flags "$@" \
-  && echo Now type \`make\' to compile. || exit 1
+  $srcdir/configure $conf_flags "$@" &&
+  echo Now type \`make\' to compile. || exit 1
 else
   echo Skipping configure process.
 fi
